sig-and-wg/wg-knative/knative-eventing-adoption.md

```diff
@@ -52,6 +52,20 @@ With Knative adoption, this model can no longer be applied due to the following:
 
 >Note: A Kyma Event type is just a logical concept. The actual Event type is a Fully Qualified Event Type composed of `application(source id) + event type + event type version`.
 
+## Knative channel naming convention
+ The Knative channel name follows this pattern: `{event-source-id}-{event-type}-{event-type-version}`. In order to avoid naming conflicts, use the following guidelines when creating the channel name: 
```

Comment:
```suggestion
 The Knative channel naming follows this pattern: `{event-source-id}-{event-type}-{event-type-version}`. In order to avoid naming conflicts, the following rules are considered when creating the channel name: 
```